frontend/src/pages/modelServing/screens/global/InferenceServiceListView.tsx

```diff
@@ -5,8 +5,9 @@ import SearchField, { SearchType } from '~/pages/projects/components/SearchField
 import { ModelServingContext } from '~/pages/modelServing/ModelServingContext';
 import { ProjectsContext } from '~/concepts/projects/ProjectsContext';
 import { getInferenceServiceDisplayName, getInferenceServiceProjectDisplayName } from './utils';
-import ServeModelButton from './ServeModelButton';
+//import ServeModelButton from './ServeModelButton';
```

Comment:
Commented code

Reply:
Will be fixed in https://github.com/opendatahub-io/odh-dashboard/pull/2154